### PR TITLE
feat(api): add log field from body

### PR DESCRIPTION
**Pull request by fsamin**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-cds#20734.0 ✘  
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20738.0 ✘  
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20740.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20740.0 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘  
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20740.1 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

File: sdk/v2_workflow_run.go [hunk 1]
```diff
@@ -34,24 +34,24 @@ type V2WorkflowRunHookRequest struct {
...
-	ProjectKey    string                 `json:"project_key" db:"project_key"`
+	ID            string                 `json:"id" db:"id" cli:"id" action_metadata:"workflow-run-id"`
+	ProjectKey    string                 `json:"project_key" db:"project_key" action_metadata:"project-key"`
 	VCSServerID   string                 `json:"vcs_server_id" db:"vcs_server_id"`
-	VCSServer     string                 `json:"vcs_server" db:"vcs_server"`
+	VCSServer     string                 `json:"vcs_server" db:"vcs_server" action_metadata:"vcs-server"`
 	RepositoryID  string                 `json:"repository_id" db:"repository_id"`
-	Repository    string                 `json:"repository" db:"repository"`
-	WorkflowName  string                 `json:"workflow_name" db:"workflow_name" cli:"workflow_name"`
+	Repository    string                 `json:"repository" db:"repository" action_metadata:"repository-identifier"`
```
Comment (sguiheux) [inline sdk/v2_workflow_run.go:42]: We should match with existing log field no?
https://github.com/ovh/cds/blob/master/sdk/log/fields.go#L37

**Review by sguiheux — CHANGES_REQUESTED** [review]

File: sdk/v2_workflow_run.go (hunk 1 above)
Comment (fsamin) [reply sdk/v2_workflow_run.go:42]: But this one is used for router logs: https://github.com/ovh/cds/blob/master/sdk/doc/route.go#L93

**Review by richardlt — APPROVED** [review]

**Pull request merged**